README.md

```diff
@@ -482,6 +482,12 @@ steps to configure (such as setting up Nomad job specifications and performing d
 ./deploy.sh
 ```
 
+### Autoscaling and Setting Spot Prices
+
+`refinebio` uses AWS Auto Scaling Groups to provide elastic capacity for large work loads. To do this, we use "Spot Requests". To find a good bid price for your instance type, use the [spot request page](https://console.aws.amazon.com/ec2sp/v1/spot/home?region=us-east-1) and then click on the **view pricing history** chart. Choose your instance type and then choose a big price that it slightly higher than the current price for your availability zone (AZ).
```

Comment:
In `... a big price that it ...`, "it" probably should be "is". 
By the way, can you also cut this line into multiple lines? It seems so long.